scipy/stats/_distribution_infrastructure.py

```diff
@@ -1150,44 +1184,32 @@ def wrapped(self, x, y, *args, **kwargs):
         dtype = np.result_type(x.dtype, y.dtype, self._dtype)
         # yes, copy to avoid modifying input arrays
         x, y = x.astype(dtype, copy=True), y.astype(dtype, copy=True)
+        all_ood = (x < low) & (y < low) | (x > high) & (y > high)
+        swap = x > y
```

Comment:
I assume `all_ood` is "all out of domain", but it took a minute!
Probably should rename `swap`, since that is not longer done.